app/assets/stylesheets/question_index.scss

```diff
@@ -0,0 +1,25 @@
+.question-body {
+    font-size: larger;
+    font-family: -webkit-body;
+  }
+
+  .button-round {
```

Comment:
Line should be indented 0 spaces, but was indented 2 spaces<br>Rule set contains (12/10) properties